docs/guides/medical.md

```diff
@@ -0,0 +1,181 @@
+
+| Cecha | Apap | Morfina |
+|-------|------|---------|
+| Maksymalny czas działania | 10 min | 30 min |
+| Opóźnienie działania | 2 min | 0.5 min |
+| Siła przeciwbólowa | 0.6 | 0.8 |
+| Zastosowanie | Ból u przytomnych osób | Duży ból lub u nieprzytomnych |
+| Efekty uboczne | Brak | Obniżone tętno |
+| Ryzyko nieprzytomności | Pomijalne | Wysokie, wystarczą 2 użycia |
+
+
+## Pierwsza pomoc
+
+Pierwszą rzeczą, którą jako medyk musisz zrobić jest zapewnienie poszkodowanemu, a przede wszystkim sobie bezpiecznego otoczenia do przeprowadzenia pierwszej pomocy. Jest to najważniejsza procedura w całym ratownictwie medycznym, ponieważ udzielanie pomocy w niebezpiecznym środowisku może spowodować uszczerbek na zdrowiu osobie udzielającej pomocy, przez co potrzebne będzie zaangażowanie jeszcze większych środków, czego chcemy uniknąć. 
+
+Następnie należy sprawdzić, czy w przypadku braku przytomności ranny jeszcze ma szansę na odratowanie. Do tej pory niektórzy uważają, że ciało leżące w nienaturalnej pozycji oznacza zgon, co nie jest prawdą. Najpewniejszą na razie metodą jest próba przeniesienia rannego. Jeżeli w menu Medicala ikona do przeniesienia rannego jest szara, czyli nieaktywna, oznacza to na 100% zgon rannego (istnieje możliwość powstania aktualizacji w niedalekiej przyszłości dającą możliwość przeniesienia martwego). Jest to istotne pod względem zaoszczędzenia czasu oraz środków medycznych, tj. bandażowanie oraz resuscytacja martwego. 
+
+Kolejna rzecz to zatamowanie krwawienia. W przypadku wielu ran na kończynach, zastosuj stazę. Tymczasowo zatamuje krwawienie, co pozwoli ci na opatrzenie głowy oraz torsu. Następnie zabandażuj kończyny i najważniejsze! – ściągnij stazy. Są one wielokrotnego użytku, dlatego warto ich pilnować, żeby ktoś inny przez przypadek ich nie ukradł. Dodatkowo z czasem stazy powodują ból w wyniku nacisku powodującego niedokrwienie kończyny oraz blokują działanie adrenaliny, morfiny oraz transfuzji krwi, jeżeli zostaną podane do zastazowanej kończyny.
+
+Po opatrzeniu ran sprawdź, czy ranny ma tętno. Jeżeli brak, wykonuj RKO do skutku. W przypadku dużej utraty krwi, przetocz krew. Po każdym wykonanym cyklu RKO sprawdź, czy pacjent odzyskał tętno. Pamiętaj, żeby nie sprawdzać go podczas wykonywania resuscytacji przez inną osobę, ponieważ otrzymasz fałszywy wynik. Tak samo kończyna zastazowana nie pokaże żadnego pulsu. Po odzyskaniu tętna, sprawdź jego wartość oraz ciśnienia krwi. Przy zbyt niskim ciśnieniu oraz tętnie, przetocz krew oraz podaj adrenalinę. Nie podawaj adrenaliny przy wysokim tętnie, ponieważ może to doprowadzić do zawału serca! Następnie zaszyj rany, na bieżąco sprawdzaj i poprawiaj funkcje życiowe do czasu wybudzenia. W momencie kiedy pacjent jest ustabilizowany, ma małą utratę krwi, ale się nie budzi, można mu podać jedno Epi, żeby się szybciej obudził.
```

Comment:
`Epi` is used only once in this page, here. Please change that, either replace it with correct word or introduce it earlier.